cmd/object-api-interface.go

```diff
@@ -97,6 +97,8 @@ type ObjectOptions struct {
 
 	WalkFilter      func(info FileInfo) bool // return WalkFilter returns 'true/false'
 	WalkMarker      string                   // set to skip until this object
+	WalkLatestOnly  bool                     // returns only latest versions for all matching objects
+	WalkAskDisks    string                   // dictates how many disks are being listed
```

Comment:
None of this is being tested right now. You never set either. We shouldn't add a bunch of code that is never hit.

Reply:
This is actually used in the batch-expire feature.

https://github.com/miniohq/minio/tree/RELEASE.2023-08-04T17-40-21Z.hotfix

Reply:
I sent it separately but it can be merged along with batch expire if you insist.

Reply:
Please add it so we can get the complete picture. I presume it needs this anyway.

Reply:
It can't yet be added since this is a dependent PR for https://github.com/minio/minio/pull/17792